envs/vagrant/cinder/group_vars/all.yml

```diff
@@ -0,0 +1,29 @@
+---
+stack_env: vagrant
+floating_ip: 10.1.1.100
+
+xtradb:
+  sst_auth_password: asdf
+
+percona:
+  replication: False
+
+nova:
+  scheduler_default_filters: AvailabilityZoneFilter,ComputeFilter
+
+cinder:
+  volume_type: device
+  volume_file: /opt/stack/cinder-volumes
```

Comment:
it makes it easier for a person running it to change the option by just updating volume_type in the .yml or via ansible --extra-vars.